tests/docker/docker-compose.yml

```diff
@@ -42,5 +42,6 @@ services:
     - minio
     volumes:
     - '${BUILD_ROOT}:${BUILD_ROOT}'
+    - '${CLUSTERS_HOST_PATH}:/host_var'
```

Comment:
thinking a bit more about this PR, it occurs to me that `RedpandaService` could be left intact and all the logic be placed here instead. So mounting the arbitrary path to `/var/lib/redpanda/`. There might be a variable that docker-compose instantiates and that can be referenced here

Reply:
that's not natively possible they have an issue for having distinct mounts per scaled container https://github.com/docker/compose/issues/4579. However, I was able to work around it and I'm now waiting for the tests to run. I think that using my WA we will be able to leave intact the `RedpandaService` to future-proof it. see [vtools PR](https://github.com/redpanda-data/vtools/pull/618/files#diff-0af5f43de37b92f214f432df31cec26fb3001cbc75f152cb0cf5f6010bfec2cdR153-R159)

Reply:
It seems ok https://buildkite.com/redpanda/vtools/builds/2294#b2ecc860-bc10-4f30-8222-875990ec03f1